spectrocloud/resource_cluster_group.go

```diff
@@ -117,9 +119,8 @@ func resourceClusterGroupCreate(ctx context.Context, d *schema.ResourceData, m i
 
 	// Warning or errors can be collected in a slice type
 	var diags diag.Diagnostics
-
-	cluster := toClusterGroup(d)
 	scope := d.Get("context").(string)
+	cluster := toClusterGroup(c, d)
```

Comment:
@SivaanandM I haven't found changes for toClusterGroup() is it pushed? 
same for the flatten function and corresponding unit test for flatten function.

Reply:
@nikchern as discussed we are not flattening cluster profiles